pkgs/development/tools/misc/readexe/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv
+, fetchFromGitHub
+, autoreconfHook
+}:
```

Comment:
```suggestion
{ 
  lib
, stdenv
, fetchFromGitHub
, autoreconfHook
}:
```

Reply:
weird, putting `stdenv` on the first line along with `lib` was a last minute change

but since when do we put the open brace on its own line?

Reply:
> weird, putting stdenv on the first line along with lib was a last minute change

Yea this was mostly what I wanted to make you aware of it it wasn't intentional.

> but since when do we put the open brace on its own line?

There is an argument that can be made that it improves diffs, but there is no clear right answer. It's up to you ultimately.

Reply:
the `{ lib, stdenv` line is very unlikely to be changed, so diffing it isn't a concern
and i like the consistent spacing and compactness of having brace on that line :)

Reply:
If you think it's best :D